src/main/java/com/amazonaws/kinesisvideo/producer/ClientInfo.java

```diff
@@ -129,4 +139,25 @@ public boolean getLogMetric() {
     public int getAutomaticStreamingFlags() {
         return mAutomaticStreamingFlags.getStreamingFlagValue();
     }
+
+
+    /**
+     * NOTE: The below getters are not supported for setting/getting in Java. These will return
+     * null to be initialized to default/null values in the JNI and C layers.
+     */
+    public long getMetricLoggingPeriod() {
+        return 0;
```

Comment:
`return mMetricLoggingPeriod` instead so we do not have to modify this in the future if we support values getting set in the future.